TeX_files/chapter01.tex

```diff
@@ -85,37 +85,39 @@ \section{Antennas}
 \label{fig:spaceantenna}
 \end{figure}
 
-An antenna is a device which can transmit electromagnetic energy into space and also can receive electromagnetic energy coming from space. An example is the parabolic dish antenna (see figure~\ref{fig:spaceantenna}), signals coming from a point are like parallel rays when it gets to the parabolic dish antenna the rays converge to a focal point called the feed from there they get processed into an electrical signal.
+An antenna is a device which can transmit electromagnetic energy into space and also can receive electromagnetic energy coming from space. An example is the parabolic dish antenna (see figure~\ref{fig:spaceantenna}), signals coming from a point are like parallel rays .When it gets to the parabolic dish antenna the rays converge to a focal point called the feed from there they get processed into an electrical signal.
 
 An antenna is a device which separately puts radiation in the desired direction. A simple antenna structure may not necessarily provide the desired characteristics of modern-day smart antenna systems where radiation characteristics can be automatically changed to maximize the reception of the signal.
 
-Other more advanced antenna systems referred to as the \textbf{smart antennas} can selectively push the radiation in the desired direction. There are two types of smart antennas which are the \textbf{adaptive array} and \textbf{switched beam systems}.
+Other more advanced antenna systems referred to as \textbf{smart antennas} can selectively steer the radiation in the desired direction. There are two types of smart antennas which are the \textbf{adaptive array} and \textbf{switched beam systems}.
 
-\textbf{Adaptive array\index{adaptive array}:} This antenna steers the beam in the direction of the user/observer while simultaneously nulling interfering signals (see figure~\ref{fig:fh06_02}).
+\textbf{Adaptive array\index{Adaptive array antenna}:} steers the beam in the direction of the user/observer while simultaneously nulling interfering signals (see figure~\ref{fig:fh06_02}).
 \begin{figure}[h]
 \centering
 \includegraphics[scale=0.3]{./graphics/fh06_02}
 \caption{Adaptive array system}
 \label{fig:fh06_02}
 \end{figure}
 
-\textbf{Switched beam antenna\index{switched beam antenna}:} has multiple beams (see figure~\ref{fig:switchedbeam}) which can be switched depending on the area an observer is situated so that a signal can be transmitted or received from that zone. Therefore, depending on the requirement of the system a beam is selected hence the name switched beam.
+\textbf{Switched beam antenna\index{Switched beam antenna}:} has multiple beams (see figure~\ref{fig:switchedbeam}) which can be switched depending on the area an observer is situated so that a signal can be transmitted or received from that zone. Therefore, depending on the requirement of the system a beam is selected hence the name switched beam.
```

Comment:
```suggestion
\textbf{Switched beam antenna\index{switched beam antenna}:} has multiple beams (see figure~\ref{fig:switchedbeam}) which can be switched depending on the area an observer is situated so that a signal can be transmitted or received from that zone. Therefore, depending on the requirement of the system a beam is selected hence the name switched beam.
```